server/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10
+
+# If PORT variable is not passed as argument during image build, it defaults to 4000
+# i.e. docker build --build-arg PORT=${ANY PORT} ...
+ARG PORT=4000
+ENV PORT=${PORT}
+
+WORKDIR /app
+COPY package*.json ./
+RUN yarn install --production
+COPY . .
```

Comment:
Does this need to be three separate layers? Could it just be
```Dockerfile
COPY . .
RUN yarn install --production
```

Reply:
Oh! Sure I guess.

I was watching a video on how to optimize Docker through caching by splitting your commands, but I am not sure if what I'm doing actually works hahaha.

I'll just join everything to simplify the Dockerfile :)

Reply:
It's nice to optimize the Docker build sometimes, but in this case it probably makes the most sense to keep the Dockefile simple